pkg/controller/etcdproxy/config.go

```diff
@@ -0,0 +1,50 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package etcdproxy
+
+import (
+	restclient "k8s.io/client-go/rest"
+)
+
+// EtcdProxyControllerConfig type is used to pass information from cli to the controller.
+type EtcdProxyControllerConfig struct {
+	// CoreEtcdOptions contains information needed to wire up ReplicaSets and the core etcd.
+	CoreEtcdOptions *CoreEtcdConfig
```

Comment:
strip Options postfix

Reply:
Fixed, but GitHub doesn't see the change for some reason.